src/mongocxx/private/libmongoc_symbols.hpp

```diff
@@ -44,6 +44,8 @@ MONGOCXX_LIBMONGOC_SYMBOL(client_pool_try_pop)
 MONGOCXX_LIBMONGOC_SYMBOL(client_set_read_prefs)
 MONGOCXX_LIBMONGOC_SYMBOL(client_set_ssl_opts)
 MONGOCXX_LIBMONGOC_SYMBOL(client_set_write_concern)
+MONGOCXX_LIBMONGOC_SYMBOL(collection_get_name)
```

Comment:
Please sort these new entries into alphabetical order with the other collection_* symbols